pkg/nodebootstrap/generate.go

```diff
@@ -0,0 +1,3 @@
+package nodebootstrap
```

Comment:
What is this for?

Reply:
ah woops i ended up making a separate PR for this